src/App.jsx

```diff
@@ -4,6 +4,7 @@ import Header from "./Components/Header";
 import Navigation from "./Components/Navigation";
 import { Route, Routes } from "react-router-dom";
 import ArticleList from "./Components/ArticleList";
+import SingleArticle from "./Components/SingleArticle";
 
 function App() {
   const [currentArticles, setCurrentArticles] = useState([]);
```

Comment:
These are only used in article list, this state would be better in the ArticleList component